src/main.rs

```diff
@@ -80,6 +80,24 @@ struct Args {
     #[clap(long, default_value_t = 15 * 60)]
     timeout_starred: u64,
 
+    /// If duplicate listen port mitigations should be enabled.
+    /// This will track ports opened by benefice to prevent
+    /// users from listening on the same port twice.
+    #[clap(long, default_value_t = true)]
+    shared_port_protections: bool,
+
+    /// The lowest listen port allowed in an Enarx.toml.
+    #[clap(long, default_value_t = 2_000)]
+    port_min: u16,
+
+    /// The highest listen port allowed in an Enarx.toml.
+    #[clap(long, default_value_t = 30_000)]
```

Comment:
just like the min, this could be set to max by default to disable and make the feature opt-in